internal/controller/utils/utils.go

```diff
@@ -1,4 +1,4 @@
-package controllers
+package utils
```

Comment:
`utils` is not a great name. should we separate this file and potentially change the package name?

Reply:
package: `controllers`
top level funcs - `controller.go`
structs in `resolvers.go`
🤔

Reply:
renamed in here: https://github.com/ngrok/kubernetes-ingress-controller/pull/343/commits/93746a75b9c816961aacdc522e6cd0f7998f5fee